app/src/features/inntektsmelding/InntektsmeldingRootLayout.tsx

```diff
@@ -46,6 +48,18 @@ export const InntektsmeldingRootLayout = () => {
         }
       >
         <Outlet />
+        {!erPåKvitteringssiden && (
```

Comment:
Usikker på om her er beste sted å legge til den. Tenker det er naturlig at den vil vises for alle sider utenom kvittering. Men roter til rotlayouten å legge til for mange unntak. Tanker?

Reply:
Kunne man heller laget en generell "navigasjonsknapper"-komponent, som rendrer "tilbake", "neste" og en avbrytknapp, avhengig av hvilken side man er på? Da får man all informasjonen på samme sted, og slipper å ha spre logikken utover slik det blir nå.